test/ffmpeg-vaapi/transcode/avc.py

```diff
@@ -0,0 +1,158 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+from .transcoder import TranscoderTest
+
+spec = load_test_spec("avc", "transcode")
+class to_avc(TranscoderTest):
+  @slash.parametrize(*gen_transcode_parameters(spec, "avc", "hwhw"))
+  @platform_tags(set(AVC_DECODE_PLATFORMS) & set(AVC_ENCODE_PLATFORMS))
+  def test_hwhw(self, case):
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      dstextension = 'h264',
+      case         = case,
+      mode         = 'hwhw',
+      mcodec       = 'h264_vaapi',
+    )
+    self.transcode()
+  
+  @slash.parametrize(*gen_transcode_parameters(spec, "avc", "hwsw"))
+  @platform_tags(AVC_DECODE_PLATFORMS)
+  def test_hwsw(self, case):
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      dstextension = 'h264',
+      case         = case,
+      mode         = 'hwsw',
+      mcodec       = 'h264',
+    )
+    self.transcode()
+  
+  @slash.parametrize(*gen_transcode_parameters(spec, "avc", "swhw"))
+  @platform_tags(AVC_ENCODE_PLATFORMS)
+  def test_swhw(self, case):
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      dstextension = 'h264',
+      case         = case,
+      mode         = 'swhw',
+      mcodec       = 'h264_vaapi',
+    )
+    self.transcode()
+
+class to_hevc(TranscoderTest):
+  @slash.parametrize(*gen_transcode_parameters(spec, "hevc", "hwhw"))
+  @platform_tags(set(AVC_DECODE_PLATFORMS) & set(HEVC_ENCODE_8BIT_PLATFORMS))
+  def test_hwhw(self, case):
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      dstextension = 'hevc',
```

Comment:
extension should be `h265` for hevc output files

Reply:
changed to h265